src/MainUpdate.elm

```diff
@@ -267,8 +276,18 @@ update msg model =
             UpdateNewSessionSubmissionIds newSubmissionIdsString ->
                 ( { model | submissionIdsInput = newSubmissionIdsString }, Cmd.none )
 
-            UpdateNewSessionColumn newColumnId ->
-                ( (updateNewSession model (\ns -> { ns | columnId = (toInt model newColumnId) })), Cmd.none )
+            UpdateNewSessionColumn newColumn ->
+                case (String.toInt newColumn) of
+                    -- a column id integer was passed to the update
+                    Ok columnIdInt ->
+                        ( (updateNewSession model (\ns -> { ns | sessionColumn = (ColumnId columnIdInt) })), Cmd.none )
+
+                    Err _ ->
+                        if newColumn == "ALL COLUMNS" then
```

Comment:
This is looks like an error when it really isn't. I think it's clearer to handle the case of "ALL COLUMNS" (possibly using an if) and then do then try to convert it to an Int.